tests/test/libs/helpers/helpers.bash

```diff
@@ -17,11 +17,11 @@ EOF
 function create_makefile () {	
   export MAKEFILES="${1}/Makefile"
   cat << EOF > "${MAKEFILES}"
-ROOT_DIR = $(pwd)/examples/simple
+ROOT_DIR = $(pwd)
 NAMESPACE = bats-test
 ICMK_VERSION ?= master
```

Comment:
Does that mean it will only test master? If yes, we should test currently checked-out revision.
If revision is set somewhere else, we should probably remove this assignment from here, if possible.

Reply:
no, we pull the current code version from PR